contrib/build-linux/appimage/Dockerfile

```diff
@@ -1,51 +1,61 @@
 # Note: we deliberately use an old Ubuntu LTS as base image.
 # from https://docs.appimage.org/introduction/concepts.html :
 # "[AppImages] should be built on the oldest possible system, allowing them to run on newer system[s]"
-FROM ubuntu:16.04@sha256:a4fc0c40360ff2224db3a483e5d80e9164fe3fdce2a8439d2686270643974632
+FROM ubuntu:18.04@sha256:9bc830af2bef73276515a29aa896eedfa7bdf4bdbc5c1063b4c457a4bbb8cd79
```

Comment:
Similar question, GA do not rely on this Dockerfile right?

Reply:
Similar to #12, it is being used in the file [[contrib/build-linux/appimage/build.sh](https://github.com/BirthdayResearch/defichain-electrum/pull/13/files#diff-f8adb7985674339e5eafb97ac5414a113204a1b76e99a40aa10ba5e2e1755b83)].